test/server.js

```diff
@@ -946,7 +946,20 @@ describe('server', function() {
         , multicast: true
       }).end()
     })
-  
+
+  })
+
+})
+
+describe('piggyback timing option', function() {
+  it('send message', function(done) {
+    var piggyBackTimeout = 42
+
+    var server = coap.createServer({ piggybackReplyMs: piggyBackTimeout })
+
+    expect(server._options.piggybackReplyMs).to.eql(piggyBackTimeout)
+    server.close()
+    done()
```

Comment:
It would be better if we test this with a `setTimeout`, and add a test also for the default.